webapp/src/main/webapp/templates/freemarker/body/contactForm/contactForm-form.ftl

```diff
@@ -36,7 +36,7 @@
         </div>
 
 
-        <#if captchaToUse == "RECAPTCHA">
+        <#if captchaToUse == "RECAPTCHAv2">
```

Comment:
Any chance to make this case insensitive? Also, I suggest to use lowercase in example.runtime.properties (including text in comments, for instance recaptchav2 and nanocaptcha

Reply:
It most certainly can be made case insensitive. I will switch to using lowercase as well.